docs/data-sources/compute_flavors.md

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Elastic Cloud Server (ECS)"
+---
+
+# hcso_compute_flavors
+
+Use this data source to get the available Compute Flavors.
+
+## Example Usage
+
+```hcl
+data "hcso_compute_flavors" "flavors" {
+  availability_zone = "cn-north-1a"
+  performance_type  = "normal"
+  cpu_core_count    = 2
+  memory_size       = 4
+}
+
+# Create ECS instance with the first matched flavor
+resource "hcso_compute_instance" "instance" {
+  flavor_id = data.hcso_compute_flavors.flavors.ids[0]
+
+  # Other properties...
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) The region in which to obtain the flavors.
+  If omitted, the provider-level region will be used.
+
+* `availability_zone` - (Optional, String) Specifies the AZ name.
+
+* `performance_type` - (Optional, String) Specifies the ECS flavor type. Possible values are as follows:
+  + **normal**: General computing
+  + **computingv3**: General computing-plus
+  + **highmem**: Memory-optimized
+  + **saphana**: Large-memory HANA ECS
+  + **diskintensive**: Disk-intensive
+
+* `generation` - (Optional, String) Specifies the generation of an ECS type. For example, **s3** indicates
+  the general-purpose third-generation ECSs. For details, see
+  [ECS Specifications](https://support.hcso.com/intl/en-us/productdesc-ecs/ecs_01_0014.html).
```

Comment:
This link is not available, Please remove it.

Reply:
Removed